dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNVscConverterInterfaceIIDM.cpp

```diff
@@ -103,6 +111,11 @@ VscConverterInterfaceIIDM::getP() {
   return getPInjector();
 }
 
+double
+VscConverterInterfaceIIDM::getQMax() {
+  return vscConverterIIDM_.getReactiveLimits<powsybl::iidm::ReactiveLimits>().getMaxQ(getP());
```

Comment:
are we sure it is getP() and not - getP() ? To be verified

Reply:
It is -getP(), I will do the correction, thanks!